pages/common/whoami.md

```diff
@@ -0,0 +1,7 @@
+# whoami
+
+> Display currently logged user
+
+- Display currently logged user id
```

Comment:
isn't the user ID a separate concept from user name? i.e. `whoami | id -u` should return the user ID, and `whoami` the user name.